client/src/components/card/styles.css

```diff
@@ -1,39 +1,47 @@
-
-.showMore{
+.showMore {
   display: grid;
-  grid-template-rows:140px 450px;
-      margin-top: 10px;
+  grid-template-rows: 7rem 26rem;
+  margin-top: 5px;
 }
-.hideAll{
+.hideAll {
   display: grid;
-  grid-template-rows:140px;
-    margin-top: 10px;
+  grid-template-rows: 7rem;
+  margin-top: 5px;
 }
-.cardDetails{
+.cardDetails {
   display: flex;
+  justify-content: center;
   border: solid 1px;
   background-color: #f77;
   border-radius: 10px;
   font-family: "ubuntu";
-  font-size: 16px;
-  width: 100%;
+  font-size: 1.2rem;
 }
 .subContainer {
-  padding: 15px;
+  padding: 1rem;
 }
 
 .imgDim {
-  width:33px;
-  height: 33px;
+  margin-top: 1rem;
+  width: 2rem;
+  height: 2rem;
+}
```

Comment:
Please add a space between each selector - see this in the last section of the file.